lib/processes/proxy/proxy.js

```diff
@@ -85,6 +95,7 @@ function emptySuccess(response) {
 function proxyCommand(url, options, response, fromPort) {
   switch (url) {
   case '/new-page':
+  case '/testium-new-page':
```

Comment:
Might be overly careful to keep support around for it. Something similar is true for `redirect: true`. I don't think anyone *should* be hitting the proxy with anything but `redirect: true`.